pkgs/top-level/python-packages.nix

```diff
@@ -6114,13 +6114,14 @@ self: super: with self; {
     IOKit = pkgs.darwin.apple_sdk_11_0.IOKit;
   };
 
-  jaxlib = self.jaxlib-build;
+  # Using the -bin version since the -source one is currently broken (see #323154)
+  jaxlib = self.jaxlib-bin;
```

Comment:
Personally, I've no preference as to whether this is implement in `jax/default.nix` or in `python-packages.nix`. Let's keep this as a source build on `linux-*`. Let's keep it at `-bin` for `darwin-*` but use `meta.unsupported` instead of `broken`

Reply:
Switched to this logic